services/bettersocial/post/createPost.js

```diff
@@ -99,7 +99,9 @@ const BetterSocialCreatePost = async (req, isAnonimous = true) => {
 
     body.tagUsers?.forEach(async (user_id) => {
       await sendMultiDeviceTaggedNotification(
-        isAnonimous ? {username: `Anonymous ${body?.anon_user_info?.emoji_name}`} : userDetail,
+        isAnonimous
+          ? {username: `${body?.anon_user_info?.color_name} ${body?.anon_user_info?.emoji_name}`}
+          : userDetail,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change seems fine as long as the `color_name` and `emoji_name` properties are always defined when the user is anonymous. If these properties can be undefined or null, you should add a fallback value to prevent potential issues.
```diff
- isAnonimous ? {username: `Anonymous ${body?.anon_user_info?.emoji_name}`} : userDetail,
+ isAnonimous
+   ? {username: `${body?.anon_user_info?.color_name || 'Unknown'} ${body?.anon_user_info?.emoji_name || ''}`}
+   : userDetail,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->